<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
<comment> Begin CVS Header </comment>
<comment>   $Source: /Volumes/Home/Users/shoops/cvs/copasi_dev/copasi/UI/CQStartValueReset.ui,v $ </comment>
<comment>   $Revision: 1.1.36.1 $ </comment>
<comment>   $Name:  $ </comment>
<comment>   $Author: aekamal $ </comment>
<comment>   $Date: 2011/01/31 17:34:08 $ </comment>
<comment> End CVS Header </comment>

<comment> Copyright (C) 2010 by Pedro Mendes, Virginia Tech Intellectual </comment>
<comment> Properties, Inc., University of Heidelberg, and The University </comment>
<comment> of Manchester. </comment>
<comment> All rights reserved. </comment>

 <class>CQStartValueReset</class>
 <widget class="QDialog" name="CQStartValueReset">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>120</width>
    <height>151</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Start Value Reset</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="QPushButton" name="mpBtnModel">
     <property name="text">
      <string>Model Value</string>
     </property>
     <property name="shortcut">
      <string/>
     </property>
     <property name="autoDefault">
      <bool>true</bool>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="mpBtnSolution">
     <property name="text">
      <string>Current Solution</string>
     </property>
     <property name="shortcut">
      <string/>
     </property>
     <property name="autoDefault">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="mpBtnRandom">
     <property name="text">
      <string>Random</string>
     </property>
     <property name="shortcut">
      <string/>
     </property>
     <property name="autoDefault">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="mpBtnCancel">
     <property name="text">
      <string>Cancel</string>
     </property>
     <property name="shortcut">
      <string/>
     </property>
     <property name="autoDefault">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="mpSpace">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>60</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <resources/>
 <connections>
  <connection>
   <sender>mpBtnCancel</sender>
   <signal>clicked()</signal>
   <receiver>CQStartValueReset</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mpBtnRandom</sender>
   <signal>clicked()</signal>
   <receiver>CQStartValueReset</receiver>
   <slot>slotRandom()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mpBtnModel</sender>
   <signal>clicked()</signal>
   <receiver>CQStartValueReset</receiver>
   <slot>slotModel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mpBtnSolution</sender>
   <signal>clicked()</signal>
   <receiver>CQStartValueReset</receiver>
   <slot>slotSolution()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
